packages/service-clients/tinylicious-client/api-report/tinylicious-client.beta.api.md

```diff
@@ -22,20 +22,20 @@ export class TinyliciousClient {
     }>;
 }
 
-// @public
+// @public @sealed
 export interface TinyliciousClientProps {
     readonly connection?: TinyliciousConnectionConfig;
     readonly logger?: ITelemetryBaseLogger;
 }
 
-// @public
+// @public @sealed
 export interface TinyliciousConnectionConfig {
     readonly domain?: string;
     readonly port?: number;
     readonly tokenProvider?: ITokenProvider;
 }
 
-// @public
+// @public @sealed
 export interface TinyliciousContainerServices {
     readonly audience: ITinyliciousAudience;
 }
```

Comment:
@ChumpChief The only interfaces I hesitated to mark `@sealed` are the `Member` and `User` interfaces here. We expect app authors to potentially extend these, right?

Reply:
Doing so probably bleeds into the protocol summary, which would be undesirable.  I think we'd instead advise customers to keep a sidekick data structure to look up extra metadata.